Samples/Samples/App.xaml

```diff
@@ -10,6 +10,21 @@
             <Style TargetType="Page" ApplyToDerivedTypes="True">
                 <Setter Property="Visual" Value="{x:Static local:App.PreferredVisual}" />
             </Style>
+            <Style TargetType="Editor">
```

Comment:
For some reason, the default text on iOS with material is invisible in text boxes. So we just make it black for this sample. Probably a real reason and might be a bad style. But, for this sample app, no need to spend much time on it. :)